pkg/status/status.go

```diff
@@ -0,0 +1,237 @@
+// Copyright 2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package status
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/cilium/cilium/pkg/lock"
+	"github.com/cilium/cilium/pkg/logging"
+	"github.com/cilium/cilium/pkg/logging/logfields"
+)
+
+const (
+	defaultInterval         = 5 * time.Second
```

Comment:
We usually store default consts in `pkg/default`.

Reply:
If the constants are localized to a package, then I think it makes more sense to keep them within that package. If a constant is utilized by multiple packages, then it should be in `pkg/default`.

Reply:
I'm fine with both approaches. Maybe keeping in `pkg/default` makes a default more visible.